pkg/scripts/kubeadm.go

```diff
@@ -28,18 +28,21 @@ var (
 
 		sudo kubeadm {{ .VERBOSE }} join \
 			--config={{ .WORK_DIR }}/cfg/master_{{ .NODE_ID }}.yaml
+		sudo find /etc/kubernetes/pki/ -name *.crt -exec chown 600 {} \;
```

Comment:
This should be `chmod` instead of `chown`.

```suggestion
		sudo find /etc/kubernetes/pki/ -name *.crt -exec chmod 600 {} \;
```

Reply:
:+1: fixed